conda/common/anaconda_client.py

```diff
@@ -119,23 +119,40 @@
 #     return re.sub(r'([./])conda([./])', r'\1api\2', url, count=1)
 
 
+def get_binstar_directory():
```

Comment:
TODO @srossross 

Maybe this is just the binstar_client import.  I don't really care where the binstar directory is or where you're keeping your information.

The real thing I need to do is `read_binstar_tokens` so I can match them up as appropriate.